### ISSUE TITLE
Using an EntityManagerDecorator yelds a fatal error

### ISSUE THREAD

**Issue by mcorteel-harel** [schmittjoh/JMSJobQueueBundle#249]
According to Doctrine best practices, I implemented an EntityManagerDecorator. But then the `jms-job-queue:run` command yielded:
```
Return value of JMS\JobQueueBundle\Command\RunCommand::getEntityManager() must be an instance of Doctrine\ORM\EntityManager, instance of App\Service\EntityManagerDecorator returned.
```
My `App\Service\EntityManagerDecorator` extends `Doctrine\ORM\Decorator\EntityManagerDecorator`, according to https://github.com/doctrine/orm/blob/2.7/lib/Doctrine/ORM/EntityManager.php#L55
This is not really documented and I'm not sure if I did it correctly, but I guess `JMS\JobQueueBundle\Command\RunCommand` should expect `Doctrine\ORM\EntityManagerInterface` instead of `Doctrine\ORM\EntityManager` for its return value.

I'll gladly provide more details if necessary.

**Comment by cklm** [comment]
you are right, the Interface should be used to ensure that Decorators work properly - did an pull-request
